src/content/docs/tutorials/map-data-to-ocsf.mdx

```diff
@@ -396,9 +403,9 @@ let $proto_nums = {
   ipv6: 41,
 }
 ocsf.connection_info = {
-  community_uid: zeek.community_id,
-  protocol_name: zeek.proto,
-  protocol_num: $proto_nums[zeek.proto].otherwise(-1),
+  community_uid: move zeek.?community_id,
+  protocol_name: move zeek.proto,
+  protocol_num: $proto_nums.get(zeek.proto, -1),
```

Comment:
Note: Depending on whether we merge https://github.com/tenzir/tenzir/pull/5128 this may need to be adapted.

```suggestion
  community_uid: move zeek.community_id?,
  protocol_name: move zeek.proto,
  protocol_num: $proto_nums[zeek.proto?]? else -1
```